index.html

```diff
@@ -1 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+    <header id="header">
+        Product landing page
+         <img id="header-img" src="dat8zlJpGUVKLUoyScZRa/9k=" alt="hello">
+         <nav id="nav-bar">
+           <a class="nav-link" href="#landing_page">#landing page</a>
+           <a class="nav-link" href="#landing_page">News</a>
+           <a class="nav-link" href="#landing_page">Contact</a>
+         </nav>
+    </header>
+    <video src="www.nopage.vo.cv" id="video"></video>
+    <form id="form" action="POST">
+      <input type="email" id='email' name="email" placeholder="enter email" value="">
+      <button type="submit"  id="submit">submit</button>
+    </form>
+    @qxopL6TpxalPHriMfxHvLa3p0qcKVKEadOEVGFOEVGMYrkklwSNWbcbO6jqeuW9KEa1tZW1NKF9uKcIVd11HUit5Nty3Ic
```

Comment:
Please attribute to this type of file on what it must do. There is no clear explanation of what the file is supposed to do in the page. I would advise you to put it semantically in Html tags :+1: